eventsapi/utils.py

```diff
@@ -0,0 +1,104 @@
+import json
+import boto3
+from functools import cache
+import py_avro_schema as pas
+from aiokafka import AIOKafkaProducer
+from aws_schema_registry.avro import AvroSchema
+from aws_schema_registry import SchemaRegistryClient, DataAndSchema
+from kafka.serializer import Serializer
+from kafka.record.legacy_records import LegacyRecordBatchBuilder
+from kafka.errors import MessageSizeTooLargeError
+from aws_schema_registry.adapter.kafka import KafkaSerializer
+from eventsapi.settings import \
+    GLUE_REGISTRY_NAME, AWS_REGION, \
+    KAFKA_BOOTSTRAP_BROKERS
+from eventsapi.models import \
+    KafkaContentLoadedV1, KafkaContentLoadFailedV1, \
+    CONTENT_LOADED_V1, CONTENT_LOAD_FAILED_V1
+
+
+@cache
+def get_schema(eventname):
```

Comment:
@rnathuji Would you also suggest moving `KafkaAvroSerializer()` to avro.py?

Reply:
:thinking: -- No, probably because it's only used by `get_kafka_producer()` and is really just a helper class for that. I happened to call it `KafkaAvroSerializer` to try and differentiate from the `KafkaSerializer` we import from `aws_schema_registry` for the "real" / Glue code path. If that causes confusion maybe we can name the class something different :man_shrugging: , but I'd keep it in the same file.